package.json

```diff
@@ -1,11 +1,14 @@
 {
   "name": "@lifeomic/chroma-react",
-  "version": "0.5.4",
+  "version": "0.0.0",
```

Comment:
Since this `version` entry is no longer a source of truth, we've taken the approach of setting it to `0.0.0` for clarity.